tests/test-sessions/test-infinispan-sessions/src/test/java/org/eclipse/jetty/server/session/remote/RemoteInfinispanTestSupport.java

```diff
@@ -86,11 +137,13 @@
             }
 
             String content = baos.toString("UTF-8");
-            _manager.getCache("___protobuf_metadata").put("session.proto", content);
+            _manager.administration().getOrCreateCache("___protobuf_metadata", (String)null).put("session.proto", content);
+            _manager.administration().getOrCreateCache("remote-session-test", (String)null);
```

Comment:
Is this creating a cache named "remote-session-test" on the infinispan server? If so, we don't need to do that because at line 170 we will create it there?

Reply:
yes but in line 170 it use the field `_name` which can have a different value. This line of code is just here to start with `standalone` configuration and create the cache `remote-session-test` at start as if we were using the file `remote-session-test.xml`

Reply:
Right, but all the remote tests pass in the name "remote-session-test" don't they?